test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -96,3 +97,8 @@ def test_pytorch_telemetry_gpu(pytorch_training, ec2_connection, gpu_only):
 @pytest.mark.parametrize("ec2_instance_type", ["c5.4xlarge"], indirect=True)
 def test_pytorch_telemetry_cpu(pytorch_training, ec2_connection, cpu_only):
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_TELEMETRY_CMD)
+
+
+@pytest.mark.parametrize("ec2_instance_type", ["p2.xlarge"], indirect=True)
```

Comment:
nice, thanks for adding. Out of curiosity, is this same change needed in other containers, or is it already present?

reason for asking - if already present, we could make this test framework agnostic.

Reply:
It is present in TF2.2 only till now.

Reply:
Perhaps then we can add the test inside `test/dlc_tests/ec2/`, kind of like smdebug? Do you anticipate this test should be required to pass for every GPU training container?

Reply:
if it currently fails for mxnet and tf1, for instance, we can skip for those images

Reply:
note: you now need to add model as a required marker.

You can decorate this function as follows:

```
@pytest.mark.model("N/A")
```